EventFilter/CSCTFRawToDigi/src/CSCTFEvent.cc

```diff
@@ -3,7 +3,7 @@
 #include <string.h>
 #include <stdexcept>
 
-unsigned int CSCTFEvent::unpack(const unsigned short *buf, unsigned int length) throw() {
+unsigned int CSCTFEvent::unpack(const unsigned short *buf, unsigned int length) noexcept(false) {
```

Comment:
hi @mrodozov - its probably best just to remove the throw() and replace it with nothing
@Dr15Jones (opinions?)